src/main/markdown/doc/latest/polymer-tutorial/introduction.md

```diff
@@ -0,0 +1,48 @@
+# Building a modern GWT app using Polymer Elements:
+
+## Introduction
+
+In this tutorial, you’ll learn how to write a **TodoList** application using [Polymer][0] [Web Components][1]. Web Components define a collection of standards allowing us to bundle markup and styles into custom HTML elements. Polymer is a JS library to make easier deal with Web Components.
+
+In this case, we want to fulfil the [Material Design][2] specification. Therefore, we’ll be using the Vaadin [gwt-polymer-elements][3] library. This library, generated by [gwt-api-generator][4], is a JsInterop wrapper for the Polymer [Iron and Paper][5] elements.
+
+Before we start, you might want to try out the [TodoList][6] application to see what we are about to build.
+
+## **Disclaimer**:
```

Comment:
IMO, it is very redundant to have the same section in 6 pages.
I think the disclaimer is very clear in the introduction page, and it is clear the position of the gwtproject related with external libraries. Repeating the disclaimer 6 times does not add more info but noise.